js/src/util/wallet.js

```diff
@@ -41,27 +43,26 @@ export class Wallet {
       if (kdfparams.prf !== 'hmac-sha256') {
         throw new Error('Unsupported parameters to PBKDF2');
       }
-      derivedKey = crypto.pbkdf2Sync(pwd, salt, kdfparams.c, kdfparams.dklen, 'sha256');
+      derivedKey = pbkdf2Sync(pwd, salt, kdfparams.c, kdfparams.dklen, 'sha256');
```

Comment:
Crypto is implemented in JS, so it will run in single a single thread anyway. The API is there for node compatibility: https://github.com/crypto-browserify/pbkdf2/blob/master/browser.js#L13